sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -22,7 +23,7 @@
 from ._result import Result
 
 
-Literal = Union[int, str, float, bool, None]
+Literal = Union[int, str, float, bool, None, timedelta]
```

Comment:
Torn on whether to try to use more specific types to limit where `timedelta` can be used. I think probably not -- even if we protect against a literal in an unexpected place, a timedelta column could show up there.

Reply:
I'm not sure what you mean by the last sentence. Is the timedelta not a literal here?

Reply:
It is, and can be passed *anywhere* we accept a `Literal`. Alternative would be for `add` to say:

```
def __add__(self, rhs: Union[Timestream, Literal, timedelta]) -> Timestream:
```

Basically, choose where to allow it. But then we can still have a timedelta arise as a itmestream (eg., `seconds_between`, so it doesn't buy us much safety.